CONTRIBUTING.md

```diff
@@ -265,6 +265,9 @@ Once that's done, just run:
 yarn test && yarn build --skip-os-packages
 ```
 
+You can get all build options using the following command :  
+`yarn build --help`
```

Comment:
super minor nit, can we do a code block if it's on a newline or inline the command?  only for consistency

lgtm after!